### ISSUE TITLE
Cleanup

### ISSUE THREAD

**Issue by NathanX-S** [OpenToontownTools/OpenLevelEditor#23]
Realizing that these are pretty minor things, but things that should still be kept of, making a checklist issue. Nor are they necessarily urgent.

**Checklist**
- [ ] Make stylefiles more sane, JSON maybe?
- [ ] Replace all uses of 0/1 with bools for better readability/pythonism
- [ ] Ensure all deprecated functions are replaced.

Please, if you feel anything should be added! Discuss! I also think that things need to be a little more compartmentalized. A 6k line file is insane and a pain.

**Comment by drewc5131** [comment]
Yeah I had json in mind for style files at some point too. Definitely wanna do that since the current ones are abhorrent. 
Just need to include an auto converter as well

**Comment by drewc5131** [comment]
I wouldnt mind moving the editor panel and the visgroups panel to a different file. Definitely would make life easier

**Comment by drewc5131** [comment]
About the replace 0/1 with true/false, you can't do that with the IntVar variables for the panel, just the way PMW functions.

Although PMW sucks so I would love to transition to QT at some point if I can get that done
